src/Forms/Statistics.cs

```diff
@@ -162,125 +158,111 @@ private void Statistics_KeyDown(object sender, KeyEventArgs e)
             {
-                string s = word.Trim();
-
+                var s = word.Trim();
                 if (s.Length > 1 && hashtable.ContainsKey(s))
                 {
-                    int hits = int.Parse(hashtable[s]);
-                    hits++;
-                    hashtable[s] = hits.ToString(CultureInfo.InvariantCulture);
+                    hashtable[s]++;
                 }
                 else if (s.Length > 1)
                 {
-                    hashtable.Add(s, "1");
+                    hashtable.Add(s, 1);
                 }
             }
         }
 
-        private static void MostUsedLinesAdd(Dictionary<string, string> hashtable, string lastLine)
+        private static void MostUsedLinesAdd(Dictionary<string, int> hashtable, string text)
         {
-            lastLine = lastLine.Trim('\'');
-            lastLine = lastLine.Replace("\"", "");
-            lastLine = lastLine.Replace("<i>", "");
-            lastLine = lastLine.Replace("</i>", ".");
-            lastLine = lastLine.Replace("<I>", "");
-            lastLine = lastLine.Replace("</I>", ".");
-            lastLine = lastLine.Replace("<b>", "");
-            lastLine = lastLine.Replace("</b>", ".");
-            lastLine = lastLine.Replace("<B>", "");
-            lastLine = lastLine.Replace("</B>", ".");
-            lastLine = lastLine.Replace("<u>", "");
-            lastLine = lastLine.Replace("</u>", ".");
-            lastLine = lastLine.Replace("<U>", "");
-            lastLine = lastLine.Replace("</U>", ".");
-            lastLine = lastLine.Replace('!', '.');
-            lastLine = lastLine.Replace('?', '.');
-            lastLine = lastLine.Replace("...", ".");
-            lastLine = lastLine.Replace("..", ".");
-            lastLine = lastLine.Replace('-', ' ');
-            lastLine = lastLine.FixExtraSpaces();
-            string[] lines = lastLine.Split('.');
-
-            foreach (string line in lines)
-            {
-                string s = line.Trim();
+            text = StripHtmlTags(text);
+            text = text.Replace('!', '.');
+            text = text.Replace('?', '.');
+            text = text.Replace("...", ".");
+            text = text.Replace("..", ".");
+            text = text.Replace('-', ' ');
+            text = text.FixExtraSpaces();
 
+            foreach (string line in text.Split('.'))
+            {
+                var s = line.Trim();
                 if (hashtable.ContainsKey(s))
                 {
-                    int hits = int.Parse(hashtable[s]);
-                    hits++;
-                    hashtable[s] = hits.ToString(CultureInfo.InvariantCulture);
+                    hashtable[s]++;
                 }
                 else if (s.Length > 0 && s.Contains(' '))
                 {
-                    hashtable.Add(s, "1");
+                    hashtable.Add(s, 1);
                 }
             }
         }
 
+        private static string StripHtmlTags(string text)

```

Comment:
perhaps use HtmlUtil.RemoveHtmlTags ?
(I also wonder if AppVoyor will be okay if you change anything now...)